### DISCUSSION TITLE
Inline Action to Button

### DISCUSSION THREAD

**Discussion by vv-ac** [filamentphp/filament#2434] [Help] [answered]
Hi all,

I have several inline actions in a table which I can style in terms of color and icon.
Is it also possible to add extraAttributes (similar to table cells) so I can make it look like a button?

This:
```
>extraAttributes(['class' => ...'])
```
does not work as it is not know to Action.

Thanks!

**Comment by danharrin** [comment, marked as answer]
`->button()`, right?

**Reply by vv-ac** [reply]
Sounds legit and works, thank you!
Do you know where this is documented?
I looked at the following sources:
- https://filamentphp.com/docs/2.x/tables/actions
- https://filamentphp.com/docs/2.x/admin/pages#actions

**Reply by R1NEJ0** [reply]
Hi @danharrin I'm trying to create a new button on edit page, "Save and Close" button. I'd like to order buttons.
Ex.
Save - My custom button - Save & create other - close.

 But I'm unable to order it.

I've been trying with '->button()' but doesn't add attributes...

**Reply by danharrin** [reply]
```php
protected function getFormActions(): array
{
    return [
        $this->getSaveFormAction(),
        // your custom form action here
        $this->getCancelFormAction(),
    ];
}
```

**Reply by R1NEJ0** [reply]
~~Hi! This work perfectly in when extends from EditRecord, but not when extends from CreateRecord...~~

~~ `Method App\Filament\Resources\FooResource\Pages\CreateFoo::getSaveFormAction does not exist.`~~

A lot of thanks!
